app/assets/stylesheets/base/_buttons.scss

```diff
@@ -75,8 +75,8 @@ button[type="submit"],
 }
 
 .btn-small {
-  padding: ($small-spacing/2);
-  font-size: 12px;
+  padding: ($small-spacing/2) !important;
```

Comment:
I do not why, but the class `.btn-small` don't make any difference when I've generated a button with: `f.submit` inside `profiles/_form`.